contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -385,11 +481,15 @@ contract BosonVoucher is IBosonVoucher, BeaconClientBase, OwnableUpgradeable, ER
      * @notice Transfers ownership of the contract to a new account (`newOwner`).
      * Can only be called by the protocol. Change is done by calling `updateSeller` on the protocol.
      *
-     * @param newOwner - the address to which ownership of the voucher contract will be transferred
+     * @param _newOwner - the address to which ownership of the voucher contract will be transferred
      */
-    function transferOwnership(address newOwner) public override(IBosonVoucher, OwnableUpgradeable) onlyRole(PROTOCOL) {
-        require(newOwner != address(0), "Ownable: new owner is the zero address");
-        _transferOwnership(newOwner);
+    function transferOwnership(address _newOwner)
+        public
+        override(IBosonVoucher, OwnableUpgradeable)
+        onlyRole(PROTOCOL)
+    {
+        require(_newOwner != address(0), "Ownable: new owner is the zero address");
```

Comment:
we could add this to the place where we have all of the revert reasons in the future ...

Reply:
Moved this and "ERC2981: royalty fee exceeds protocol limit" to BosonConstants.sol.